test/contconic.jl

```diff
@@ -503,5 +503,106 @@ function contconictest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64))
         end
     end
 
+    if MOI.supportsproblem(solver, MOI.ScalarAffineFunction, [(MOI.ScalarAffineFunction{Int}, EqualTo{Int}), (MOI.ScalarAffineFunction{Int}, EqualTo{Rational{Int}}), (MOI.VectorVariablewiseFunction, MOI.PositiveSemidefiniteConeTriangle), (MOI.VectorVariablewiseFunction, MOI.SecondOrderCone)])
```

Comment:
I don't think we should expect solvers to implement ``MOI.ScalarAffineFunction{Int}`` and ``EqualTo{Int}`` in addition to ``MOI.ScalarAffineFunction{Float64}`` and ``EqualTo{Float64}`` unless they have a special reason to do so. It imposes a lot of extra work on the interface because they have to be maintained as a separate class of constraints (the constraint references have a different type).
None of the SDP solvers we have connected support rational arithmetic, so I wouldn't expect them to support `EqualTo{Rational{Int}}` either.